klt-cert-manager/config/default/manager_config_patch.yaml

```
@@ -8,3 +8,10 @@ spec:
     spec:
       containers:
       - name: manager
+        resources:
+          limits:
+            cpu: 500m
```

I have second thoughts on this :/ 
At the end of the day, the cert-manager doesn't need much CPU since it only creates certificates. 
```suggestion
            cpu: 100m
```